app/middlewares/user.js

```diff
@@ -34,3 +38,13 @@ exports.checkOrder = [
     .customSanitizer(value => value && value.toUpperCase())
     .isIn([ascOrder, descOrder])
 ];
+
+exports.checkBoughtAlbumsPermission = (req, res, next) => {
+  if (
+    (req.userPayload.role === default_role && req.userPayload.id === parseInt(req.params.userId)) ||
+    req.userPayload.role === admin_role
```

Comment:
When you have some large conditions is nice separate this in different constants, like:
```
const isAdmin = req....;
if (isAdmin || other) {
..code
}
```